### ISSUE TITLE
JS bundles not found after SSR build + render

### ISSUE THREAD

**Issue by adebree** [feloy/angular-cli-universal#2]
This is the result of the dist directory after an aot browser build (ng build -aot), an aot ssr build (ng build -aot -app ssr) and an rendered module (via renderModuleFactory() in node)

<img width="202" alt="screen shot 2017-07-25 at 12 50 53" src="https://user-images.githubusercontent.com/478872/28568974-11932012-7138-11e7-95a1-2fc908d32234.png">

But in the index.html that was generated expects the JS bundles in the same directory as the generated as the index.html is:

<img width="885" alt="screen shot 2017-07-25 at 12 54 20" src="https://user-images.githubusercontent.com/478872/28569054-81dc04d8-7138-11e7-9f8f-70b976e303fa.png">

What would be the best approach? Copy'ing the bundles from the browser build to the root of dist?

**Comment by feloy** [comment]
Yes, I think that would be a solution. Or place the browser/index.html used by node index.js in another directory and place the created files in browser directory.
